subprojects/plugins/src/test/kotlin/org/gradle/kotlin/dsl/plugins/precompiled/PrecompiledScriptPluginTest.kt

```diff
@@ -259,6 +259,8 @@ class PrecompiledScriptPluginTest : AbstractPluginTest() {
 
                     $pluginManagementBlock
 
+                    enableFeaturePreview("STABLE_PUBLISHING")
```

Comment:
❓ does the test fail without this?

Reply:
Yes, it does because in the currently used Gradle snapshot (as in `master`) the publishing block still uses `@DeferredConfigurable` by default. The code I changed causes the block to be evaluated which then fails these builds with a "cannot change publishing block after it has been evaluated" message.

Reply:
Ok. So if I understand it correctly, we will be able to remove those feature flags once we are on a new wrapper built on top of latest  Gradle master, right?

Reply:
Yep, once the change is merge there. To do so, we need this change merged here first, though. Cyclic dependencies are so much fun... 😉

Reply:
Indeed :)